internal/boxcli/writer/writer.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+package writer
+
+import "io"
+
+type DevboxIOWriter struct {
+	W     io.Writer
+	Quiet bool
+}
+
+func (d DevboxIOWriter) Write(p []byte) (int, error) {
+	if !d.Quiet {
+		n, err := d.W.Write(p)
+		if err != nil {
+			return n, err
+		}
+		if n != len(p) {
+			return n, io.ErrShortWrite
+		}
+	}
```

Comment:
what was the resolution of the discussion for having `--quiet` silence the output versus put it in Stderr?

Reply:
I think we can do both, I can pass in stderr as an io writer, so that devbox outputs are written on Stderr only. Also have -q silence the output.